docs/gallery/External_Resources.py

```diff
@@ -63,15 +69,21 @@
 # when referencing a key. Let's start by creating a new key.
 
 # Let's start with a new instance of ExternalResources
-
 er = ExternalResources(name='new_example')
 
 # The add_key method will create a new key
-er.add_key(key_name='human')
-er.add_key(key_name='gorilla')
+er.add_key(key_name='Homo sapien')
+er.add_key(key_name='Mus Musculus')
 
 # The get_key method will return the key object of the key_name.
-key_object = er.get_key(key_name='gorilla')
+key_object = er.get_key(key_name='Mus Musculus')
+
+# We can use the key_object to reference keys.
+er.add_ref(container=data, field='', key=key_object, resource_name='NCBI_Taxonomy',
```

Comment:
Please explain that this is useful for associating an existing key with an additional entity. 
You can use resource_name='ITIS', resource_uri='https://www.itis.gov/', entity_id='180366', entity_uri='https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=180366' as a second entity for Mus musculus.